hw2/03-count.js

```diff
@@ -1,3 +1,19 @@
 // Add your code here
 
+const input = document.getElementById('userInput');
+const textField = document.getElementById('text');
+const highlightedText = document.getElementById('highlightedText');
+
+const handleKeyDown = function handleUserKeyDown(event) {
+  if (event.keyCode === 13) {
+    const searchTerm = input.value.trim();
+    const searchRegex = new RegExp(`\\b${searchTerm}\\b`, 'gi');
+    const searchContent = textField.innerHTML;
+    const highlightedContent = searchContent.replace(searchRegex, '<span style="background-color: yellow;">$&</span>');
+    highlightedText.innerHTML = highlightedContent;
+    textField.style.display = 'none';
+    highlightedText.style.display = 'block';
+  }
+};
```

Comment:
![corey count](https://github.com/CoreyPatrickPDX/frontend-homework/assets/38127011/b0bf9ab7-e3ae-4dca-ab87-e58c2ef9c583)


There is a bug in the implantation of this exercise. A space is not a word. So spaces in the paragraph should not be highlighted when a space is entered in the input box. Please update the code to consider this case.